components/layout_2020/flexbox/layout.rs

```diff
@@ -721,12 +721,14 @@ impl FlexContainer {
         };
 
         // Implement "unsafe" alignment. "safe" alignment is handled by the fallback process above.
-        let resolved_align_content = self.config.resolve_reversable_flex_alignment(
```

Comment:
Implement "unsafe" alignment. "safe" alignment is handled by the fallback process above.

"let flex_wrap_is_reversed = flex_context.config.flex_wrap_is_reversed;"